### PR TITLE
docs: fix styling

### PR DESCRIPTION
**Pull request by juliodialpad**
## Description

Fixed general styling issues

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [ ] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [x] Ensure all `gulp` scripts successfully compile.
 - [ ] Update, remove, or extend all affected documentation.
 - [x] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).

### CODE REVIEW

**Comment by braddialpad** [comment]
Very Excite!

File: docs/.vuepress/theme/assets/less/dialtone-docs.less [hunk 1]
```diff
@@ -175,15 +166,20 @@ nav a.router-link-active {
...
-  h2 {
-    &:hover a.header-anchor {
-      opacity: 1;
-    }
+  max-width: 90rem;
+  margin: 0 auto;
+  padding: 2rem 2.5rem;
+}
+
+a.header-anchor {
```
Comment (braddialpad) [inline docs/.vuepress/theme/assets/less/dialtone-docs.less:174]: Something up with the links on landing pages

**OLD**
Screenshot 2022-11-30 at 4 17 41 PM

**NEW**
Screenshot 2022-11-30 at 4 17 33 PM

File: docs/.vuepress/theme/assets/less/dialtone-docs.less [hunk 2]
```diff
@@ -93,9 +95,8 @@ span.DocSearch-Button-Keys {
 //  $   SEARCH BUTTON
 //      Override search button with dialtone styles
 //  ----------------------------------------------------------------------------
-nav a.router-link-active {
-  font-weight: var(--fw-bold);
-  text-decoration: none;
+a.router-link-active {
+  font-weight: var(--fw-semibold);
```
Comment (braddialpad) [inline docs/.vuepress/theme/assets/less/dialtone-docs.less:99]: The links in sidebar/header seem to have a higher weight than they did on the old site, but that might be intended?

Comment (braddialpad) [reply docs/.vuepress/theme/assets/less/dialtone-docs.less:99]: Links on front page have underlines where they didn't on the old site.

File: docs/.vuepress/theme/assets/less/dialtone-docs.less [hunk 3]
```diff
@@ -175,15 +166,20 @@ nav a.router-link-active {
...
-    opacity: 0;
-  }
-  h2 {
-    &:hover a.header-anchor {
-      opacity: 1;
-    }
+  max-width: 90rem;
+  margin: 0 auto;
+  padding: 2rem 2.5rem;
+}
```
Comment (braddialpad) [inline docs/.vuepress/theme/assets/less/dialtone-docs.less]: overall page width is tighter than before

File: docs/.vuepress/theme/components/Sidebar.vue [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <template>
   <ul
     v-if="sidebarItems.length"
-    class="d-p24"
+    class="d-t64 d-b0 d-ps-fixed d-p24 d-pb96 d-h100vh d-of-y-auto"
```
Comment (braddialpad) [inline docs/.vuepress/theme/components/Sidebar.vue:4]: scrollbar seems to always be there even if the screen has enough height to show all items

**Review by braddialpad — COMMENTED** [review]
It is a bit hard to compare, because I'm not sure if some of the styles on the old site are incorrect due to the default theme overriding them. I have pointed out things I have noticed which are not necessarily wrong, just want to confirm that they are intended.

Also not entirely sure of the scope of this PR. Should every single style on every single page of the site look correct at this point?

This is sweet, looking forward to it.

**Comment by juliodialpad** [comment]
> It is a bit hard to compare, because I'm not sure if some of the styles on the old site are incorrect due to the default theme overriding them. I have pointed out things I have noticed which are not necessarily wrong, just want to confirm that they are intended.
> 
> Also not entirely sure of the scope of this PR. Should every single style on every single page of the site look correct at this point?
> 
> This is sweet, looking forward to it.

Yes, I want to have all the styling "corrected" in this PR. I think the site should look almost the same here than before the custom theme, after that we can start improving styling.

File: docs/.vuepress/theme/assets/less/dialtone-docs.less (hunk 1 above)
Comment (juliodialpad) [reply docs/.vuepress/theme/assets/less/dialtone-docs.less:174]: Good catch! I'll add d-td-none to all the links

File: docs/.vuepress/theme/assets/less/dialtone-docs.less (hunk 2 above)
Comment (juliodialpad) [reply docs/.vuepress/theme/assets/less/dialtone-docs.less:99]: > The links in sidebar/header seem to have a higher weight than they did on the old site, but that might be intended?

I think the font is different now, @francisrupert can you help us to decide which styling is correct, please?

Comment (juliodialpad) [reply docs/.vuepress/theme/assets/less/dialtone-docs.less:99]: > Links on front page have underlines where they didn't on the old site.

Fixed

File: docs/.vuepress/theme/assets/less/dialtone-docs.less (hunk 3 above)
Comment (juliodialpad) [reply docs/.vuepress/theme/assets/less/dialtone-docs.less]: Fixed, changed to 99rem, that was the previous value.

File: docs/.vuepress/theme/components/Sidebar.vue (hunk 4 above)
Comment (juliodialpad) [reply docs/.vuepress/theme/components/Sidebar.vue:4]: Can it be your OS configuration? I see the scrollbar always on dialpad.design and not on the new site.

**Dialpad.design**
Screenshot 2022-12-01 at 8 26 36 p m

**Deploy preview**
Screenshot 2022-12-01 at 8 27 31 p m

**Local**
Screenshot 2022-12-01 at 8 28 09 p m

File: docs/.vuepress/theme/assets/less/dialtone-docs.less (hunk 3 above)
Comment (braddialpad) [reply docs/.vuepress/theme/assets/less/dialtone-docs.less]: Lookin better. I think text paragraphs had a different width on the current site

Example

Screenshot 2022-12-02 at 1 25 52 PM

VS


Screenshot 2022-12-02 at 1 25 47 PM

File: docs/.vuepress/theme/components/Sidebar.vue (hunk 4 above)
Comment (braddialpad) [reply docs/.vuepress/theme/components/Sidebar.vue:4]: Yeah I think its fine

**Review by braddialpad — APPROVED** [review]
Pre approving this since I'm not noticing anything that looks particularly off anymore (other than the paragraph thing I mentioned).

File: docs/.vuepress/theme/assets/less/dialtone-docs.less (hunk 3 above)
Comment (juliodialpad) [reply docs/.vuepress/theme/assets/less/dialtone-docs.less]: Yes, I noticed that, will add that constraint here too.

**Pull request merged**